.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: php
 
 php:
-  - 5.6
-  - 7.0
```

Comment:
Mezzio skeleton requires 7.1: https://github.com/mezzio/mezzio-skeleton/blob/master/composer.json#L43

Should this be pointed out in the doc, as Codeception instead supports 5.6 and 7.0?